pkgs/tools/games/gamemode/default.nix

```diff
@@ -14,34 +14,22 @@
 
 stdenv.mkDerivation rec {
   pname = "gamemode";
-  version = "1.6.1";
+  version = "1.7";
 
   src = fetchFromGitHub {
     owner = "FeralInteractive";
     repo = pname;
     rev = version;
-    sha256 = "sha256-P00OnZiPZyxBu9zuG+3JNorXHBhJZy+cKPjX+duZrJ0=";
+    sha256 = "sha256-DIFcmWFkoZOklo1keYcCl6n2GJgzWKC8usHFcJmfarU=";
   };
 
   outputs = [ "out" "dev" "lib" "man" "static" ];
 
   patches = [
-    # Run executables from PATH instead of /usr/bin
-    # See https://github.com/FeralInteractive/gamemode/pull/323
-    (fetchpatch {
-      url = "https://github.com/FeralInteractive/gamemode/commit/be44b7091baa33be6dda60392e4c06c2f398ee72.patch";
-      sha256 = "TlDUETs4+N3pvrVd0FQGlGmC+6ByhJ2E7gKXa7suBtE=";
-    })
-
-    # Fix loading shipped config when using a prefix other than /usr
-    # See https://github.com/FeralInteractive/gamemode/pull/324
-    (fetchpatch {
-      url = "https://github.com/FeralInteractive/gamemode/commit/b29aa903ce5acc9141cfd3960c98ccb047eca872.patch";
-      sha256 = "LwBzBJQ7dfm2mFVSOSPjJP+skgV5N6h77i66L1Sq+ZM=";
-    })
-
     # Add @libraryPath@ template variable to fix loading the PRELOAD library
     ./preload-nix-workaround.patch
+    # Do not install systemd sysusers configuration
+    ./no-install-systemd-sysusers.patch
```

Comment:
Hmm, yeah we definitely don't want this in the Nix package. The NixOS module uses the packaged polkit policies to let you run `cpugovctl` & `gpuclockctl` without root. Although, maybe we should consider refactoring that to require your user to be a part of the gamemode group, so not everyone can run them? (not for this PR though)

Either way, I think a meson option should be added upstream to disable it so we could get rid of the patch in a future update.

Reply:
I'm not exactly sure how to approach the process of adding a group, giving it appropriate permissions and then somehow prompt the user to add themselves to it, so I'll leave this part as-is.